qlty-cli/src/commands/coverage/complete.rs

```diff
@@ -10,9 +10,13 @@ use qlty_coverage::{
     publish::{Plan, Planner, Settings},
     token::load_auth_token,
 };
-use serde_json::Value;
 use std::time::Instant;
 
+#[derive(Debug, Default, Clone)]
+pub struct CompleteResult {
+    pub url: String,
+}
```

Comment:
A more Rust-y way to do this would be to apply the newtype pattern using `enum`: https://www.lurklurk.org/effective-rust/newtype.html

However, it seems like just using a String would also be fine. I would reach for the newtype pattern when there's more stuff to do with the data m